met-api/tests/unit/api/test_feedback.py

```diff
@@ -0,0 +1,64 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to verify the Feedback API end-point.
+
+Test-Suite to ensure that the /Feedbacks endpoint is working as expected.
+"""
+import json
+
+from tests.utilities.factory_scenarios import TestJwtClaims
+from tests.utilities.factory_utils import factory_auth_header, factory_feedback_model
+
+
+def test_feedback(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that an feedback can be POSTed."""
+    claims = TestJwtClaims.public_user_role
+
+    feedback = factory_feedback_model()
+    to_dict = {
+        'rating': feedback.rating,
+        'comment_type': feedback.comment_type,
+        'comment': feedback.comment
+    }
+    headers = factory_auth_header(jwt=jwt, claims=claims)
+    rv = client.post('/api/feedbacks/', data=json.dumps(to_dict),
+                     headers=headers, content_type='application/json')
+    assert rv.status_code == 200
+
```

Comment:
Can you please assert the structure of the json?Like assert a few fields is in json
That will really help if we change something in the data model and UI has to change, the test broken will indicate that the UI has to change